site/content/docs/user/known-issues.md

```diff
@@ -39,6 +40,12 @@ description: |-
 * [IPv6 Port Forwarding](#ipv6-port-forwarding) (docker doesn't seem to implement this correctly)
 * [Couldn't find an alternative telinit implementation to spawn](#docker-init-daemon-config)
 * [Fedora](#fedora) (various)
+* [Failed to get rootfs info](#failed-to-get-rootfs-info--stat-failed-on-dev)
+
+## Troubleshooting Kind
+
+If the cluster fails to create, try again with the `--retain` option (preserving the failed container),
+then run `kind export logs` to export the logs from the container to a temporary directory on the host.
```

Comment:
do you mind address one thing only and focus on the rootfs problem by now?

Reply:
Sure, if you prefer it to be split. I just figured that adding a short paragraph to the top of the page contributed information that I wanted to include in my answer (i.e. where to find the error) but was also common to many of the other known issues in the page.

Reply:
my concern is that it doesn't look like a section in its own
Maybe we can reword the description block in lines 8-21 of this file ?